Sources/ParseSwift/Types/ParseError.swift

```diff
@@ -12,12 +12,25 @@ import Foundation
  An object with a Parse code and message.
  */
 public struct ParseError: ParseType, Decodable, Swift.Error {
+    /// The value representing the error from the Parse Server.
     public let code: Code
+    /// The text representing the error from the Parse Server.
     public let message: String
+    /// An error value representing a custom error from the Parse Server.
+    public let otherCode: Int?
+    public init(code: Code, message: String) {
```

Comment:
Based on my comment earlier https://github.com/parse-community/Parse-Swift/issues/164#issuecomment-862269923

I still don't see the need for developers to create errors on the client-side and call it a "Parse Error". If a developer creates an error in CloudCode, it should decode with the current additions you have in this PR and doesn't need this init